datalad/distributed/ora_remote.py

```diff
@@ -883,6 +874,39 @@ def _verify_config(self, gitdir, fail_noid=True):
             self.storage_host, self.store_base_path, self.ria_store_url = \
                 verify_ria_url(self.ria_store_url, url_cfgs)
 
+        else:
+            # for now still accept the configs, if no ria-URL is known, but
+            # issue deprecation warning:
+            host = _get_gitcfg(gitdir,
+                               'annex.ora-remote.{}.ssh-host'.format(name)) or \
+                   self.annex.getconfig('ssh-host')
+            self.storage_host = None if host == '0' else host
```

Comment:
I think moving the deleted comment about '0' (line 916) to here could avoid some head scratching from readers, but it's obviously not very important given this code is set to be removed.

Reply:
True. Intended to move it, but obv. forgot about it. Thx.